src/Cake.NuGet/NuGetPackageInstaller.cs

```diff
@@ -228,7 +228,13 @@ private FilePath GetNuGetPath()
                 arguments.Append("-Prerelease");
             }
 
-            arguments.Append("-ExcludeVersion -NonInteractive -NoCache");
+            // NoCache
+            if (definition.Parameters.ContainsKey("nocache"))
```

Comment:
can you sugest where? I'm having a hard time navigating your test codebase! 
SO MANY TESTS!